src/current/cockroachcloud/quickstart.md

```diff
@@ -99,7 +105,97 @@ The code sample uses the connection string stored in the environment variable `J
     BUILD SUCCESSFUL in 3s
     2 actionable tasks: 2 executed
     ~~~
+
+</section>
+<section class="filter-content" markdown="1" data-scope="node">
```

Comment:
```suggestion

<section class="filter-content" markdown="1" data-scope="node">
```
For readability / maintenance.

Reply:
for some reason this is also creating a bug in the markdown rendering below it